src/components/DropdownList/DropdownListItem.svelte

```diff
@@ -0,0 +1,56 @@
+<script lang="ts">
+  export let selected: boolean;
+  export let disabled: boolean = false;
+  export let title: string | undefined = undefined;
+  export let styleJustifyContent: "flex-end" | undefined = undefined;
```

Comment:
I think this is unused

Reply:
a9f1f958d0916129719dfb2bfcbba15569c38475